core/src/main/java/com/klaytn/caver/wallet/keyring/KeyringFactoryWrapper.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2020 The caver-java Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.klaytn.caver.wallet.keyring;
+
+import org.web3j.crypto.CipherException;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Representing a KeyringFactoryWrapper class which wraps all of static methods of KeyringFactory
+ */
+public class KeyringFactoryWrapper {
+
+    /**
+     * Creates a KeyringFactoryWrapper instance
+     */
+    public KeyringFactoryWrapper() {
+    }
+
+    /**
+     * Generates a keyring instance.
+     * @return Keyring
```

Comment:
I saw that KeyringFactory's return also says Keyring.
Can you change the return value in more detail at the other PR? (`KeyringFactoryWrapper.java` and `KeyringFactory.java`)

Reply:
Hi @jimni1222  
@sirano11 and I discussed about this issue and we decide to update return type of comments of `KeyringFactory` and `KeyringFactoryWrapper`.

Here is the related commit ["Update comments of methods of KeyringFactory"](https://github.com/klaytn/caver-java/pull/215/commits/284188bdec3acc84a568216f6d17e616cf24e6a7) I pushed :)

Thanks for your feedback.